Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/ReadBlob.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Text;
+using Azure.Storage.Blobs;
+using Azure;
+#pragma warning disable 1591
+namespace Frends.AzureBlobStorage.ReadBlob
+{
+    public class AzureBlobStorage
+    {
+        /// <summary>
+        /// Read a single file to Azure blob storage using connection string or SAS Token.
+        /// [Documentation](https://github.com/FrendsPlatform/Frends.AzureBlobStorage/tree/main/Frends.AzureBlobStorage.ReadBlob)
+        /// </summary>
+
+        public static Result ReadBlob([PropertyTab] Source source, CancellationToken cancellationToken)
+        {
+            var data = ReadBlobContentAsyncTest(source, cancellationToken);
+            return new Result(data.ToString());
+        }
+
+        private static string ReadBlobContentAsyncTest(Source source, CancellationToken cancellationToken)
```

Comment:
This method is not Async, so remove that from the name. Also, please remove Test from the name, since this should not be a test method.